protocol/simplex-messaging.md

```diff
@@ -639,52 +645,45 @@ ok = %s"OK"
 
 Both the recipient and the sender can use TCP or some other, possibly higher level, transport protocol to communicate with the server. The default TCP port for SMP server is 5223.
 
-By default, the client and server should use the protocol presented below, that does not depend on a centralized certificate authority.
+For scenarios when meta-data privacy is critical, it is recommended that:
+- clients and servers communicating over Tor network,
+- establish a separate connection for each SMP queue,
+- send noise traffic (using PING command).
```

Comment:
```suggestion
For scenarios when meta-data privacy is critical, it is recommended that clients and servers:
- communicate over Tor network,
- establish a separate connection for each SMP queue,
- send noise traffic (using PING command).
```

Reply:
it's not exactly right... the client does all that, the server can only be deployed as Tor service though, right?...